core/evancore.vue.libs/src/components/forms/json/json.ts

```diff
@@ -51,7 +51,7 @@ export default class FormControlJSONComponent extends mixins(EvanControlComponen
   isInvalidJSON = false;
 
   created(): void {
-    this.stringValue = JSON.stringify(this.value);
+    this.stringValue = this.value ? JSON.stringify(this.value) : '{}';
```

Comment:
```suggestion
    this.stringValue = JSON.stringify(this.value) ?? '{}';
```
This should work, shouldn't it? :)

Reply:
nope, value is an empty string by default, stringify will result in `""` (string including two quotes).

Reply:
Aye, maybe default values should be `null` instead